### DISCUSSION TITLE
global atoms cannot be accessed from subtrees with Provider?

### DISCUSSION THREAD

**Discussion by YeThor** [pmndrs/jotai#2301] [Q&A]
Let's say there is an atom `atomA`, `atomB`, `atomC` and a provider with `customStore` 

```
const SomeAtomProvider = () => {
  const customStore = createStore();

  customStore.set(atomA, 0);
  customStore.set(atomB, 0); 

  return ( 
       <Provider  store={customStore}>
            <ChildComponent /> 
       </Provider>;
  )
};
```

In this case, `<ChildComponent/>` can access only `atomA`, `atomB` but not `atomC`? 
All components below the provider ONLY see `customStore`?  
Using `useAtom(atomC, {store: getDefaultStore()})` just works but I wonder it is the only & best solution for this 🤔

**Comment by dai-shi** [comment]
Yes, that's how it works. And `{ store }` is the valid solution, and only solution with pure Jotai. (By pure it means it's a declarative atom model and atom model doesn't have a concept of store.)

Having that said, we are exploring a new solution in ecosystem. Please check out https://github.com/jotaijs/jotai-scope.
